packages/liveblocks-node/src/client.ts

```diff
@@ -48,6 +53,46 @@ type ThreadParticipants = {
   participantIds: string[];
 };
 
+type Permission = "room:write" | "room:read" | "room:presence:write";
```

Comment:
it's missing the comment permissions:
https://github.com/liveblocks/liveblocks/blob/main/packages/liveblocks-core/src/protocol/AuthToken.ts#L5

Is it also possible to bring in from @liveblocks/core?

Reply:
It's true that we have types for `room:presence:write`, `comments:write` and `comments:read` but the decoder used to validate permissions in our backend only checks for the following:

```
"room:write" | "room:read" | "room:presence:write"
```

See: https://github.com/liveblocks/liveblocks-cloudflare/blob/main/packages/liveblocks-cloudflare/src/api/lib/decoders.ts#L32-L49